src/frontend/Felles/Input/TekstInput/EnsligTextArea.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { ChangeEvent } from 'react';
 import styled from 'styled-components';
 import { FamilieTextarea, IFamilieTextareaProps } from '@navikt/familie-form-elements';
+import { SkjemaelementFeilmelding } from 'nav-frontend-skjema';
 
-export const EnsligTextArea: React.FC<IFamilieTextareaProps> = styled(FamilieTextarea)`
+const StyledFamilieTextArea: React.FC<IFamilieTextareaProps> = styled(FamilieTextarea)`
     white-space: pre-wrap;
     word-wrap: break-word;
     max-width: 60rem;
     .typo-element {
         padding-bottom: 0.5rem;
     }
 `;
+
+interface Props {
+    value: string;
+    onChange: (event: ChangeEvent<HTMLTextAreaElement>) => void;
+    label: string;
+    maxLength: number;
+    erLesevisning: boolean;
+    feilmelding?: string;
+}
+
+export const EnsligTextArea: React.FC<Props> = ({
+    value,
+    onChange,
+    label,
+    maxLength,
+    erLesevisning,
+    feilmelding,
+}) => {
+    return (
+        <>
+            <StyledFamilieTextArea
+                value={value}
+                onChange={(e) => onChange(e)}
+                label={label}
+                maxLength={maxLength}
+                erLesevisning={erLesevisning}
+            />
+            <SkjemaelementFeilmelding>{feilmelding}</SkjemaelementFeilmelding>
+        </>
+    );
+};
```

Comment:
Kan vurdere dette? Hvis du også synes koden er lettlest 😅 
```suggestion
type Props = { feilmelding: string | undefined } & IFamilieTextareaProps;

export const EnsligTextArea: React.FC<Props> = ({ feilmelding, ...propsUtenFeilmelding }) => {
    return (
        <>
            <StyledFamilieTextArea {...propsUtenFeilmelding} />
            <SkjemaelementFeilmelding>{feilmelding}</SkjemaelementFeilmelding>
        </>
    );
};
```

Reply:
Den er igrunn fin, men også litt "skummel" hvis man legger til props uten å følge de helt igjennom. Med TS får vi vel "feil" et eller annet sted, så det hjelper på, men i `good-old-days` var det vrient å lese en slik klasse og forstå hva i alle dager som befinner seg inni `propsUtenFeilmelding` :D 

For min del: begge deler funker fint. :)

Reply:
> men i good-old-days var det vrient å lese en slik klasse og forstå hva i alle dager som befinner seg inni propsUtenFeilmelding

Alt som ikke er feilmeldinger, duh 😏 

Har heller ingen sterke meninger om hva som er best. Men vi kan kanskje uansett ha:

```
type Props = { feilmelding: string | undefined } & IFamilieTextareaProps;
```